pkg/monitor/intervalcreation/node.go

```diff
@@ -192,6 +201,62 @@ func eventsFromKubeletLogs(nodeName string, kubeletLog []byte) monitorapi.Interv
 	return ret
 }
 
+// eventsFromOVSVswitchdLogs returns the produced intervals.  Any errors during this creation are logged, but
+// not returned because this is a best effort step
+func eventsFromOVSVswitchdLogs(nodeName string, ovsLogs []byte) monitorapi.Intervals {
+	nodeLocator := monitorapi.NodeLocator(nodeName)
+	ret := monitorapi.Intervals{}
+
+	scanner := bufio.NewScanner(bytes.NewBuffer(ovsLogs))
```

Comment:
Looks good. Do you think we could use strings to iterate using range loop as we know the logs will always be in new line instead of buffer?

Reply:
Do you mean converting the log file byte to a string, splitting it on newlines, then iterating on range over that array?

I suspect that would be much slower and more memory intensive than the byte buffer scanning approach, if it works the way I suspect.  Looking around this seems to be the recommended approach for iterating lines out of file contents in a byte array. Let me know if I've missed anything though.